.github/workflows/deploy-s3-bucket.yml

```diff
@@ -0,0 +1,51 @@
+name: deploy-s3-bucket-to-s3
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  compile-and-upload:
+    runs-on: ubuntu-latest
+    steps:
+      - name: clone
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+
+      - name: cache-node-modules
+        uses: actions/cache@v1
+        id: cache-node-modules
+        with:
+          path: node_modules
+          key: node-modules-${{ hashFiles('yarn.lock') }}
```

Comment:
this should be changed by package-lock.json

Reply:
indeed, thanks !